FhirToDataLake/test/Microsoft.Health.AnalyticsConnector.Core.UnitTests/Jobs/DicomToDatalakeSchedulerServiceTests.cs

```diff
@@ -209,7 +209,7 @@ public async Task GivenBrokenMetadataStoreReturnNullWhenGetTriggerLeaseEntity_Wh
 
             using var tokenSource = new CancellationTokenSource();
 
-            tokenSource.CancelAfter(TimeSpan.FromSeconds(2));
+            tokenSource.CancelAfter(TimeSpan.FromSeconds(3));
```

Comment:
Maybe we can add some comment for this delay (why 3 works) so that people writing new test cases can avoid this kind of error?

Reply:
Added some comments.